variants/ET_Board/pins_arduino.h

```diff
@@ -0,0 +1,20 @@
+// #include "pinmap.h"
+
```

Comment:
please check `variants/esp32/pins_arduino.h` for proper pin header. You need to have a few defines plus proper declaration of I2C, SPI and ADC pins. You can leave the Dx pins as defines.